File: Arrays/One Dimensional Arrays/pairSum.py
arr = [1, 2, 3, 4, 5]
sum = 5

def func(arr):
    l = len(arr)
    for i in range(l):
        for j in range(i + 1, l):
            if (arr[i] + arr[j] == sum):
                print(f"{arr[i], arr[j]}")

func(arr)

File: Arrays/One Dimensional Arrays/pairSum.cpp
#include <bits\stdc++.h>
using namespace std;

int func(int arr[], int n, int sum){
    for (int i = 0; i < n; i++){
        for (int j = i + 1; j < n; j++){
            if (arr[i] + arr[j] == sum){
                cout << arr[i] << "," << arr[j] << endl;
            }
        }
    }
}

int main(){
    int n;
    cin >> n;

    int arr[5] = {1, 2, 3, 4, 5};
    int sum = 5;

    func(arr, n, sum);
    return 0;
}
